static/tests/frontend/specs/interface_buttons.js

```diff
@@ -63,21 +63,28 @@ describe('Test the behavior of the interface buttons: Mute, Video Disable, Enlar
 
       this.timeout(60000);
 
-      expect(chrome$("video").css("width")).to.be("160px");
```

Comment:
This is an unrelated test failure that appeared randomly. It was getting back a value of `160.25px` for whatever reason. I'm just gonna round to the nearest integer and hope for the best going forward.